edx_credentials_themes/static/mitpe/css/mitpe.certificate.style-rtl.css

```diff
@@ -508,9 +508,9 @@ body {
 .accomplishment-rendering {
   text-align: center;
   margin: 2%;
-  border: 1px solid #cdd7db;
+  border: 1px solid #a0a0a0;
```

Comment:
That looks like a big jump to my eyes. I didn't check the actual color swatches, though.

Reply:
All of these grays are starting to look the same to me. I changed this to one that's slightly closer.

Reply:
Jk, changed it back.